tasks.py

```diff
@@ -1,11 +1,14 @@
 from invoke import Collection
 
-from environments.surfpol import environment
+from environments.surfpol import create_configuration_and_session
 from elastic.tasks import setup, create_snapshot, load_repository, restore_snapshot
 from deploy import prepare_builds, build, push, deploy
 from test import test
 
 
+environment, session = create_configuration_and_session(use_aws_default_profile=False)
```

Comment:
Fabric loads configurations in a similar way. It then sends any configuration to the bastion including passwords. That way the bastion does not need to store secrets, which makes security more robust.